.gitlab-ci.yml

```diff
@@ -20,18 +15,13 @@ fmt:
   stage: test
   timeout: 1 hours
   script:
-    - rustup component add rustfmt
-    - cargo fmt -- --check
+    - rustup component add rustfmt --toolchain nightly
+    - cargo +nightly fmt -- --check
 
 test:
   image: paritytech/ci-linux:105b919b-20210617
   stage: test
   timeout: 1 hours
-  only:
-    refs:
-      - branches
-    changes:
-      - ./**/*.{rs,toml,lock}
   script:
     - cargo test --all --all-features
```

Comment:
Same as above: perhaps `--all-targets`?

Reply:
That's completely unrelated to my PR! 😭 But sure i will add that...

Reply:
bfb0a36